discord_bots/utils.py

```diff
@@ -1822,3 +1823,59 @@ async def command_autocomplete(interaction: Interaction, current: str):
                         )
                     )
     return result
+
+
+def del_player_from_queues_and_waitlists(
+    session: sqlalchemy.orm.Session, player_id: int, *args: str
+) -> list[Queue]:
+    queues_del_from_by_id: dict[str, Queue] = {}
+    if args:
+        # can be a mix of queue ordinals or names
+        conditions = [
+            or_(
+                Queue.ordinal.in_(args),
```

Comment:
this fails (and already does fail in current prod code) 
`!del na 2 ctf` -> Exception thrown because the ordinal needs to be numeric

I suggest:
```
queue_ordinals = [x for x in args if all(char in "0123456789" for char in x)]
# ...
Queue.ordinal.in_(queue_ordinals),
```

Reply:
Updated the PR to reflect this change, but I simplified the above a bit to:
```python
>>> s = ('ctf',  '1', '23', '4.5', '-1', '06')
>>> [x for x in s if x.isdigit()]
['1', '23', '06']
```

Reply:
10² 10₂ ٢ 𝟜
all of those are digit too :/

Reply:
Ah, what a gotcha... In that case I'll go with the method you mentioned above. Alternatively, you could also try casting each arg to an int, and then catch the ValueError like `add` does -- the logic should definitely be consolidated into a single function, but I'll leave that to you in a future pr ;)